### PR TITLE
Support for Serverless >= 3.x and EXPRESS Workflow

### PR DESCRIPTION
**Pull request by a-marcel**
supports serverless framework >= 3.0 (tested with 3.25.1)
implement express workflow support

#20 - can someone involved in this ticket test this implementation please ?

Thanks

### CODE REVIEW

**Comment by codetheweb** [comment]
Happy to merge after someone tests it :)

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ custom:
 Although not neccessary, it's strongly recomended to add the folder with the downloaded step function executables to `.gitignore`.  By default, this path is `./.step-functions-local`.
 
 The plugin binds to port 8083, this cannot be changed.
```
Comment (codetheweb) [inline README.md:43]:
```suggestion
```

**Review by codetheweb — APPROVED** [review]

**Comment by walleXD** [comment]
Given @codetheweb approved the PR, @a-marcel can we merge it in?

Would be awesome to do a release with serverless v3 support

Thanks for all the great work @codetheweb & @a-marcel! 🙏🏽

**Comment by a-marcel** [comment]
I don't have the permission merge it into the codetheweb repo.

**Comment by walleXD** [comment]
@a-marcel I am hitting a very interesting problem. My starting state loads up and the lambda runs locally but it does not transition to the next state and eventually times out


```
[Serverless Step Functions Local] 2023-03-31 09:29:19.500: arn:aws:states:us-east-1:101010101010:execution:DataExtractorMachine:02afe082-0c6d-4680-a4fe-9fb5333d60bf : {"Type":"LambdaFunctionFailed","PreviousEventId":4,"LambdaFunctionFailedEventDetails":{"Error":"Lambda.SdkClientException","Cause":"Unable to execute HTTP request: Request did not complete before the request timeout configuration."}}
[Serverless Step Functions Local] 2023-03-31 09:29:19.504: arn:aws:states:us-east-1:101010101010:execution:DataExtractorMachine:02afe082-0c6d-4680-a4fe-9fb5333d60bf : {"Type":"ExecutionFailed","PreviousEventId":5,"ExecutionFailedEventDetails":{"Error":"Lambda.SdkClientException","Cause":"Unable to execute HTTP request: Request did not complete before the request timeout configuration."}}
```

**Comment by walleXD** [comment]
> @a-marcel I am hitting a very interesting problem. My starting state loads up and the lambda runs locally but it does not transition to the next state and eventually times out
> 
> ```
> [Serverless Step Functions Local] 2023-03-31 09:29:19.500: arn:aws:states:us-east-1:101010101010:execution:DataExtractorMachine:02afe082-0c6d-4680-a4fe-9fb5333d60bf : {"Type":"LambdaFunctionFailed","PreviousEventId":4,"LambdaFunctionFailedEventDetails":{"Error":"Lambda.SdkClientException","Cause":"Unable to execute HTTP request: Request did not complete before the request timeout configuration."}}
> [Serverless Step Functions Local] 2023-03-31 09:29:19.504: arn:aws:states:us-east-1:101010101010:execution:DataExtractorMachine:02afe082-0c6d-4680-a4fe-9fb5333d60bf : {"Type":"ExecutionFailed","PreviousEventId":5,"ExecutionFailedEventDetails":{"Error":"Lambda.SdkClientException","Cause":"Unable to execute HTTP request: Request did not complete before the request timeout configuration."}}
> ```

My pipelines have no issues when I deploy them. But with the plugin, it's stuck at the first state and does not transition to the next one

**Comment by a-marcel** [comment]
@walleXD : This plugin only simulates a local environment for your step functions. A working deployment has nothing to do with it. 

Not sure exactly what your error message means, but it looks like that you configured a pipeline and one lambda function is not answering in time or at all. 

Does the lambda functions work individually ?

**Comment by walleXD** [comment]
The lambdas do work individually when ran locally. I have also deployed the step function into the cloud and it works w/o any issue. 

With the local step function setup, the first lambda runs and then it keeps on waiting as the state does not transition to the next one to trigger the next lambda. Eventually it times out

**Review by AlexsandroBezerra — APPROVED** [review]

**Comment by bernard-fanet-shift** [comment]
Can you please merge this PR?

**Comment by codetheweb** [comment]
Released in v0.5.0.

**Comment by bernard-fanet-shift** [comment]
Thank you very much.

**Pull request merged**